### PR TITLE
feat: hey app

### LINKED ISSUE #13594: [CAL-3095] HEY Calendar
using the ics sync

<sub>[CAL-3095](https://linear.app/calcom/issue/CAL-3095/hey-calendar)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
- also fixed ics folder mismatch (needs testing)
- added hey app store entry
- ICS functionality needs testing (have no HEY account)

CleanShot 2024-02-08 at 11 15 24@2x

### CODE REVIEW

**Comment by RodEsp** [comment]
Hi @PeerRich and crew 👋🏼 
I have a Hey account and would love to help out with this. 
I started QA'ing what I could, notes below.

Also happy to help with the development, just starting to find my way around the app now though.

QA Notes
---
App shows up:
❌ http://localhost:3000/getting-started/connected-calendar
✅ http://localhost:3000/apps/categories/calendar
✅ http://localhost:3000/apps
App Page: http://localhost:3000/apps/hey
✅ Looks good, screenshots are current, similar to other app pages
Installation:
❌ http://localhost:3000/apps/hey/setup is blank (don't see any erros in browser console, network tab, or in logs from `yarn dev`)

File: packages/app-store/hey/lib/CalendarService.ts
Comment (joeauyeung) [inline packages/app-store/hey/lib/CalendarService.ts:1]: If there are going to be more apps that are similar to the ics feed. Should we make something along the lines of a `icsFeedCalendarService` class that takes in an integration name & credential?

**Comment by keithwillcode** [comment]
Moving to 4.0 because of current priorities

**Comment by keithwillcode** [comment]
Closing due to inactivity. Please reopen if work continues.

**Pull request closed without merging**